### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for document in "Azure Signin dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1380]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to document and observe the name property using inspect.

**Actual Result:**
Name property is empty for the document..

**Narrator behaviour:**
Tab
Narrator is not announcing the value. Value information for the document.

**NVDA behaviour:**
edit  read only
blank

**Expected Result:**
Name property should be simple and meaningful.
Name property for the document should be "Azure Sign in".
Narrator should  announce the value.value for the document .

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![azure signin-document-no name](https://user-images.githubusercontent.com/22956130/35681438-6bacd5c0-0783-11e8-9a64-b8eb39c9ac17.jpg)
![azure signin-document-no name-narrator](https://user-images.githubusercontent.com/22956130/35681445-6eb72572-0783-11e8-91a1-d4156a1f98df.jpg)

**Comment by Eskibear** [comment]
already shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.6.0.
Issue still exists
![1380-not fixed](https://user-images.githubusercontent.com/22956130/39797099-e9f24038-5377-11e8-99fe-5acab6689a78.jpg)

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can still see the property in the `Value` field
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

![image](https://user-images.githubusercontent.com/2351748/46598169-b8347f00-cb15-11e8-82e9-46bc56a427ad.png)

**Comment by jdneo** [comment]
Can show `name ` on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0
![image](https://user-images.githubusercontent.com/6193897/47149981-bf267300-d307-11e8-8242-967bb528fbe9.png)

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749857
